### DISCUSSION TITLE
How to find/create a database for CEA modelling?

### DISCUSSION THREAD

**Discussion by KQEFEM** [architecture-building-systems/CityEnergyAnalyst#3311] [Q&A]
Hi All, 
I am trying to model districts in Austria (primarily of at most 12-24 buildings) but was wondering how I might find/create an approriate database for the simulation based on Austrian regualtions.

From what I can see so far there are only Swiss and Singapore databases but I am unsure how I would build my own. 

In addition, is there any way of getting more accurate estimates on things like building typology outside of manually selecting the year and standard?

Best 
Kieran

**Comment by lilaleu** [comment]
Hey Kieran!

Without knowing much better, there is "only" Switzerland and Singapore as far as I can see from documentation and also tutorials. In my understanding, creating the database tailored for your use case is the toughest part about your analysis. Therefore you need much more data about your building structure e.g. "Energieausweise" (German certificates for the energetic standard of your building). Or you refer to standard profiles from other tools and transfer them into CEA (My idea first for my use case, but far to extensive for my needs). Often the true value of these building tools lies in the quality and precision of their standard profiles for the buildings. I think, if you want to have more precise estimations from CEA you'll need to change the CH database to your needs. Maybe this blog-entry can help you with that: 
https://www.cityenergyanalyst.com/blog-posts/2020/04/23/2020-4-23-working-with-cea-databases-part-3

Since Switzerland is still western Europe I'll consider the CH database as suitable for my analysis in Germany. Maybe if the building data is not too crucial for you, that might do it for Austria too? 

Would be interesting to hear more experienced user's perspective about that.

Best regards 
lilaleu

**Reply by ululele** [reply]
Hi,
Yes, you can edit all the values in the database editor.  In my case I have created a new standard and looked up for values in books and databases. 

The only problem I have is to adjust the costs for the conversion like PV, Boiler etc.  @lilaleu How did you do that? Since I am working with a side in Germany too, you might have similar values. 

Best Regards :)

**Reply by lilaleu** [reply]
I'd love to help here. But I am not so far yet. Only thing I guess to know (:D) is that the costs for the system refer to the specific costs defined in your supply system database which is created/imported into your scenario, once you have executed the archetypes mapper. You'll find the database table here: Path to project --> to scenario --> \inputs\technology\assemblies. The SUPPLY table contains the specific costs. For an individual project that might work, to simply manipulate those and run the cost-function. How the root SUPPLY-database can be manipulated: no idea so far...

**Reply by lilaleu** [reply]
cea/databases/CH/assemblies

Thats the part in your CEA folder for general changes, I suppose. Haven't tried if that works, only came across the path.
